### PR TITLE
Added new test and quick fix for withdrawal request test

### PR DESCRIPTION
**Pull request by bilalmajid-deriv**
image
image

### CODE REVIEW

File: cypress/e2e/full/cashierLegacy/paymentAgentDepositScreenLoad.cy.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        derivApp.commonPage.mobileLocators.header.hamburgerMenuButton().click()
+        derivApp.commonPage.mobileLocators.sideMenu.sidePanel().within(() => {
+          derivApp.commonPage.mobileLocators.sideMenu.cashierButton().click()
+          derivApp.commonPage.mobileLocators.sideMenu.depositButton().click()
+        })
+        cy.c_loadingCheck()
+      } else {
+        derivApp.commonPage.desktopLocators.header.cashierButton().click()
+        cy.c_loadingCheck()
+      }
```
Comment (abuzar-deriv) [inline cypress/e2e/full/cashierLegacy/paymentAgentDepositScreenLoad.cy.js:64]:
```suggestion
      if (screenSize == 'small') {
        derivApp.commonPage.mobileLocators.header.hamburgerMenuButton().click()
        derivApp.commonPage.mobileLocators.sideMenu.sidePanel().within(() => {
          derivApp.commonPage.mobileLocators.sideMenu.cashierButton().click()
          derivApp.commonPage.mobileLocators.sideMenu.depositButton().click()
        })
      } else {
        derivApp.commonPage.desktopLocators.header.cashierButton().click()
      }
      cy.c_loadingCheck()
```

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]
Minor change.

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by maliheh-deriv — APPROVED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**